src/runtime/manifest-parser.pegjs

```diff
@@ -1591,5 +1715,7 @@ eolWhiteSpace "a group of new lines (and optionally comments)"
   / spaceChar* eol eolWhiteSpace?
 eolPlusWhiteSpace "eol plus trailing whitespace"
   = eolWhiteSpace? whiteSpace?
+multiLineSpace "at least one whitespace and/or eol (and optionally comments)"
```

Comment:
eolPlusWhiteSpace meant to play similar role - it also allows comments and line breaks, even if they are a bit different today can we unify?

Reply:
Yes, I just noticed this last night. I think I'll use the `eolPlusWhitespace` definition but call it `multiLineSpace` - I originally didn't use the eolPlus version because I mistakenly thought it _required_ a newline.